.github/workflows/python-client.yml

```diff
@@ -37,21 +39,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/workflows/go-setup
-      - uses: actions/setup-python@v2
+      - uses: ./.github/workflows/python-tests
         with:
           python-version: '3.8.10'
```

Comment:
```suggestion
          python_version: '3.8.10'
```

Reply:
I went the other direction with this and made everything `python-version` so it's in line with what the `actions/setup-python@v2` workflow wants.